sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -121,29 +121,6 @@ class WholeStageCodegenSuite extends QueryTest with SharedSQLContext {
     assert(ds.collect() === Array(("a", 10.0), ("b", 3.0), ("c", 1.0)))
   }
 
-  test("cache for primitive type should be in WholeStageCodegen with InMemoryTableScanExec") {
```

Comment:
Not clear to me why this test doesn't apply anymore.

Reply:
It was verifying that InMemoryTableScanExec was inside of a WholeStageCodegen phase, but after this change InMemoryTableScanExec no longer supports codegen so the test is invalid.  The ColumnarToRowExec is what will be in the codegen section instead.